cypress/integration/main_spec.coffee

```diff
@@ -141,14 +141,22 @@ describe "Main", ->
       cy.get("aside#sidebar")
         .should("be.visible")
 
-    it "has table of contents and goes to 0.19.0", ->
-      cy.get("aside#article-toc")
-        .should("be.visible")
-        .wait(2000) # allows menuspy to load and set the menu links
-        .contains("0.19.0")
-        .click()
-      cy.url()
-        .should('include', '#0-19-0')
+    it "has a populated table of contents", ->
+      if Cypress.config('baseUrl').includes('localhost')
+        cy.get("aside#article-toc")
+          .should("be.visible")
+          .get(".toc-item")
+          .should("have.length", 5)
+        cy.url()
+          .should("match", /.+#\d+-\d+-\d+/)
+      else
+        cy.get("aside#article-toc")
+          .should("be.visible")
+          .wait(2000) # allows menuspy to load and set the menu links
```

Comment:
There'a always a better way to handle this rather than use `wait(2000)` and without seeing what's happening, I'm not sure why this is necessary.

Cypress will automatically retry for 4 seconds in the first place.

You could move this into an assertion or as part of the `contains(...)`

```js
cy.get("aside#article-toc").contains("0.19.0", { timeout: 10000 })

// or

// wait up to 10 seconds for both assertions to pass
cy.get("aside#article-toc", { timeout: 10000 }).should("be.visible").and("contain", "0.19.0")
```